ebuild-writing/misc-files/patches/text.xml

```diff
@@ -109,5 +109,282 @@ from the <c>vim</c> patch tarball:
 
 </body>
 </section>
+
+<section>
+<title>Clean Patch Howto</title>
+<body>
+
+<p>
+"Clean patch" does not refer to the patch itself (as in the changes it makes to
+the source code). It refers to all the metadata that exists in the patch to
+make it "maintainable".
+</p>
+
+</body>
+
+<subsection>
+<title>Why</title>
+<body>
+
+<p>
+This may take more effort "up front", but the amount of effort that it saves
+for everyone else in the future more than makes up for it. This refers to other
+distributions or upstream maintainers who read the patch, or future Gentoo
+maintainers/developers. By keeping all patches "clean", people can quickly and
```

Comment:
`Gentoo maintainers/developers` does this need to be separated? Aren't package maintainers called maintainers regardless of their status?

Reply:
Yep, "maintainers" is enough here.